frontend/src/router.ts

```diff
@@ -37,6 +37,15 @@ export const router = new Router({
       path: "/sound-editing/:taskId/quality-check/feedback",
       component: () => import("@/views/QCSubmissionForm.vue")
     },
+    {
+      path: "/listen/:fileName",
+      component: () => import("@/views/Listen/ListenLayout.vue"),
+      children: [
+        {
+          path: "", component: () => import("@/views/Listen/Listen.vue")
```

Comment:
Can we put this view in the root of the `/views` folder? Now it looks like an artificial folder.

Reply:
Sure, I will put it to the root of /view